tests/test_modeling_tf_dpr.py

```diff
@@ -123,8 +123,6 @@ def prepare_config_and_inputs(self):
             type_vocab_size=self.type_vocab_size,
             is_decoder=False,
             initializer_range=self.initializer_range,
-            # MODIFY
```

Comment:
Deleted all those #MODIFY statements at the cost of having to initialize the `pooler_output = None` as explained in the other comment